pygeoapi/provider/csv_.py

```diff
@@ -94,12 +99,24 @@ def _load(self, startindex=0, limit=10, resulttype='results',
         with open(self.data) as ff:
             LOGGER.debug('Serializing DictReader')
             data_ = csv.DictReader(ff)
+            data_list = list(data_)
+            fields = self.fields
+
+            # ===========================================
+            # Added for CQL Filter Expression evaluation
+
+            if filter_expression is not None:
```

Comment:
it is recommended and more pythonic to just have:

```python
if filter_expression:
```

Reply:
done